src/Exercise06AI.ipynb

```diff
@@ -140,8 +138,13 @@
     "            beta: int = Exercise06AI.LIMIT,\n",
     "        ):\n",
     "            key = Exercise04AI.get_key(board, depth)\n",
+    "            if self.stats[-1].get(\"cache_tries\", 0) == 0:\n",
```

Comment:
Evtl. auslagern (s. vorheriger Kommentar)

Reply:
Fixed in d4e37ec.